tests/sysauth/sssd.pm

```diff
@@ -52,14 +46,16 @@ sub run {
     );
 
     # for sle 12 we still use and support python2
-    push @test_subjects, 'python-pam'         if is_sle('<15');
+    if (is_sle('<15')) {
+        add_suseconnect_product('sle-module-legacy');
+        push @test_subjects, 'python-pam';
+    }
```

Comment:
just thinking: would it be more clear if you had wrapped the next line in else as it seems associated to this *if*

Reply:
Just, it is exclusive. Nice one!